Styles/style.css

```diff
@@ -121,7 +133,9 @@ body {
 
 @media only screen and (max-width: 420px) {
```

Comment:
Have the max-width of 767px instead of 420px. Some phones have larger displays and it'll also help the tablet users if they open the website.